docs/sources/datasources/prometheus/_index.md

```diff
@@ -143,39 +115,27 @@ To connect the Prometheus data source to Amazon Managed Service for Prometheus u
 
 If you run Grafana in an Amazon EKS cluster, follow the AWS guide to [Query using Grafana running in an Amazon EKS cluster](https://docs.aws.amazon.com/prometheus/latest/userguide/AMP-onboard-query-grafana-7.3.html).
 
-### Configure exemplars
+## Exemplars
+
+Exemplars are unique to Prometheus. Exemplars associate higher-cardinality metadata from a specific event with traditional time series data. See [Introduction to exemplars](https://grafana.com/docs/grafana/latest/fundamentals/exemplars/) in Prometheus documentation for detailed information on how they work.
```

Comment:
I'm not sure this is technically correct. Exemplars might work with Mimir, and though this is just a gut suggestion, I think it's an important nuance to confirm.

Reply:
checking with Brendan